src/test/java/org/schemaspy/model/CatalogTest.java

```diff
@@ -0,0 +1,29 @@
+package org.schemaspy.model;
+
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CatalogTest {
```

Comment:
My proposal is to use the brilliant [EqualsVerifier](http://jqno.nl/equalsverifier/) that makes those tests more or less a oneliner.